requirements.txt

```diff
@@ -4,6 +4,8 @@ pytest>=3.5.0
 pytest-cov>=2.5.1
 pytest-mock>=1.9.0
 Pygments>=2.2.0
+pre-commit>=1.10.5
+yamllint>=1.11.1
```

Comment:
Please also add `isort` to the list; it's not part of python's stdlib

Reply:
isort is already in the requirements file, see at the top.

Reply:
Oops, my bad